languagetool-language-modules/fr/src/main/java/org/languagetool/rules/fr/FrenchRepeatedWordsRule.java

```diff
@@ -42,7 +42,7 @@ public FrenchRepeatedWordsRule(ResourceBundle messages) {
   
   @Override
   protected String getMessage() {
-    return "Ce mot est déjà apparu. Vous pouvez utiliser un synonyme pour rendre le texte plus intéressant, à moins que la répétition ne soit intentionnelle.";
+    return "Ce mot est déjà apparu. Un synonyme peut raviver l'intérêt pour votre texte, excepté si la répétition est intentionnelle.";
   }
```

Comment:
Thanks. The original sentence was an automatic translation. 

"raviver l'intérêt pour votre texte" sounds strange to me. "dans votre text" or "de vostre text"?

Reply:
"excepté" makes sense referring to "utilitser", but not to "raviver".

Reply:
`Ce mot est déjà apparu. Utilisez un synonyme pour apporter plus de variété à votre texte, excepté si la répétition est intentionnelle.`

How about that? The current version sounds like "You can make people more interested in your text, except when the repetition was intentional" (which introduces a slight bump in logical coherence)

Reply:
Thanks, Udo. I will use your message.

Reply:
We could also say 'Ce mot est déjà apparu. Employez un synonyme pour que votre texte devienne plus attrayant, excepté si la répétition est intentionnelle.' It emphasizes the point of view of the reader, so he will like more the text.